Src/Fido2/Objects/DevicePublicKeyAuthenticatorOutput.cs

```diff
@@ -1,86 +1,85 @@
 ﻿#nullable disable
+namespace Fido2NetLib.Objects;
 
 using System;
 using Fido2NetLib.Cbor;
 
-namespace Fido2NetLib.Objects
+
+public sealed class DevicePublicKeyAuthenticatorOutput
 {
-    public sealed class DevicePublicKeyAuthenticatorOutput
+    // https://w3c.github.io/webauthn/#sctn-device-publickey-attestation-calculations
+    internal readonly byte[] _dpkAuthDataPrefix = Convert.FromHexString("64657669636520626f756e64206b6579206174746573746174696f6e2073696700ffffffff");
```

Comment:
make internal static to avoid allocations per instance... or use compiler optimized ReadOnlySpan<byte> trick.

```sharp
internal static ReadOnlySpan<byte> _dpkAuthDataPrefix => new byte[] {
   100, 101, 118, 105, 99, 101, 32, 98, 111, 117, 110, 100, 32, 107, 101, 121, 32, 97, 116, 
   116, 101, 115, 116, 97, 116, 105, 111, 110, 32, 115, 105, 103, 0, 255, 255, 255, 255
 };
 ```